tests/clients/github/test_githubclient.py

```diff
@@ -6,14 +6,17 @@
 # organization: fakeForTest2
 
 from unittest import TestCase
+import string
+from gevent import sleep
+from os import path
 
 from jumpscale.loader import j
+from tests.base_tests import BaseTests as base
 
 
 class GithubClientTest(TestCase):
     def setUp(self):
         super().setUp()
-
         self.instance_name = j.data.random_names.random_name()
 
         self.username = "tfttesting"
```

Comment:
I don't prefer to set them on the test, I think use them from environment variables would be better.